model/task/task.go

```diff
@@ -226,6 +222,14 @@ type Task struct {
 	ExecutionTasks        []string `bson:"execution_tasks,omitempty" json:"execution_tasks,omitempty"`
 	LatestParentExecution int      `bson:"latest_parent_execution" json:"latest_parent_execution"`
 
+	// Used to indictate what task to stepback to if needed.
+	LastFailingStepbackTaskId string `bson:"last_failing_stepback_task_id,omitempty" json:"last_failing_stepback_task_id"`
```

Comment:
May make sense to just have *StepbackInformation in the task instead of each field individually (I think it needs to be a pointer for omitempty to work but worth checking)